components/DescriptionList/src/index.css

```diff
@@ -0,0 +1,66 @@
+.denhaag-description-list {
+  color: var(--denhaag-description-list-color);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  font-family: var(--denhaag-description-list-font-family);
+  font-size: var(--denhaag-description-list-font-size);
+  font-weight: var(--denhaag-description-list-font-weight);
+  line-height: var(--denhaag-description-list-line-height);
+  margin-block-end: 0;
+  margin-block-start: 0;
+}
+
+.denhaag-description-list__caption {
+  color: var(--denhaag-description-list-caption-color);
+  font-family: var(--denhaag-description-list-caption-font-family, var(--denhaag-description-list-font-family));
+  font-size: var(
+    --denhaag-description-list-caption-font-size,
+    calc(var(--denhaag-description-list-font-size, 1em) * 1.33)
+  );
+  font-weight: var(--denhaag-description-list-caption-font-weight, bold);
+  line-height: var(--denhaag-description-list-caption-line-height, var(--denhaag-description-list-line-height));
+  margin-block-start: 0;
+  margin-block-end: var(--denhaag-description-list-caption-margin-block, 1em);
```

Comment:
I think this token name is misleading considering how it's applied. It's called `margin-block`, but it's only applied to `margin-block-end`

Reply:
@bddjong I refactored it in a new branch, because of a merge conflict issue with the main branch: https://github.com/nl-design-system/denhaag/pull/738